src/nvme/fabrics.c

```diff
@@ -992,3 +999,8 @@ char *nvmf_hostid_from_file()
 {
 	return nvmf_read_file(nvmf_hostid_file, NVMF_HOSTID_SIZE);
 }
+
+char *nvmf_hostsymname_from_file()
+{
+	return nvmf_read_file(nvmf_hostsymname_file, NVMF_HOSTSYMNAME_SIZE);
```

Comment:
Not sure if I want to have yet anouther file in /etc/nvme; shouldn't we rather use the json configuration here?

Reply:
Good point. JSON tends to be way simpler to parse.

Reply:
We're already asking people ton configure the hostid and hostnqn as files under /etc/nvme. So for consistency it makes sense to have the symbolic name added in the same manner. If you prefer that it be done through the json parser I have no objection.